playbooks/vars/repositories.yml

```diff
@@ -9,6 +9,10 @@ repositories:
     path: github.com/osism/ansible-collection-services
     repo: https://github.com/osism/ansible-collection-services
     branch: main
+  ansible-playbooks:
+    path: github.com/osism/ansible-playbooks
+    repo: https://github.com/osism/ansible-playbooks
+    branch: main
```

Comment:
Why is this required?

Reply:
It is not needed actually. Is a relic from testing.